types/src/connectors/configuration.ts

```diff
@@ -0,0 +1,14 @@
+import { WebCrawlerConfigurationType } from "../connectors/webcrawler";
+
+export type ConnectorConfiguration = WebCrawlerConfigurationType | null;
+
+export type ConnectorConfigurations = {
+  webcrawler: WebCrawlerConfigurationType;
+  notion: null;
+  slack: null;
```

Comment:
slack technically has a configuration as per file below but we don't set it here because we don't expect a configuration at connector creation

Reply:
yeah with default values for now

Reply:
the endpoint does not expect it basically or the type system will make the endpoint expect it if we set it here